packages/grpc-js/src/client.ts

```diff
@@ -1,3 +1,20 @@
+/*
+ * Copyright 2019 gRPC authors.
```

Comment:
It would be really awesome if these comment blocks where marked @license:
http://usejsdoc.org/tags-license.html

Reply:
I would accept such pull request :-)